src/main/java/org/jabref/preferences/FilePreferences.java

```diff
@@ -37,7 +37,7 @@ public class FilePreferences {
     private final BooleanProperty createBackup = new SimpleBooleanProperty();
     private final ObjectProperty<Path> backupDiretory = new SimpleObjectProperty<>();
 
-    public FilePreferences(String user,
+    public FilePreferences(String userAndHost,
```

Comment:
Single responsibility? Please use separate variables instead and differentiate between the vars in FilePreferences and InternalPreferences if you want to use it for different purposes.

Reply:
I renamed the variables to match the existing (!!!) content. Separatiin can be done in a follow up PR.

Reply:
In other words, calixtus can do the separation in a follow up pr?

Reply:
The renaming is now in [`7025d45` (#9910)](https://github.com/JabRef/jabref/pull/9910/commits/7025d45292e73f833b4154e8d683547aaf9e0df9). I think, the combination is OK, because it is "only" needed for the directory and the stitching together should be done as early as possible.

Nevertheless, some changes will be necessary to prevent restart of JabRef when the owner name is changed. I made a small beginning at [`cc40eaf` (#9910)](https://github.com/JabRef/jabref/pull/9910/commits/cc40eaf81f690a3b3e44bbeb0335c11916db451a).